data/json/monsters/nether.json

```
@@ -1702,5 +1702,52 @@
     "looks_like": "mon_fish_brown_trout",
     "fear_triggers": [ "FIRE" ],
     "flags": [ "HEARS", "WARM", "SWIMS", "AQUATIC", "NO_BREATHE", "WATER_CAMOUFLAGE", "ARTHROPOD_BLOOD", "GOODHEARING", "ANIMAL" ]
+  },
+  {
+    "id": "mon_nuckalavee",
+    "type": "MONSTER",
+    "name": { "str": "nuckalavee" },
+    "description": "A grotesque giant centaur like beast, with a skinless human torso grafted to an equally skinless horse.  Thick toxic smoke leeches from its two mouths as it affixes you with a gaze of pure malevolence.",
```

Wrong [adjective ordering](https://dictionary.cambridge.org/grammar/british-grammar/adjectives-order).
```suggestion
    "description": "A giant grotesque centaur-like beast, with a skinless human torso grafted to an equally skinless horse.  Thick toxic smoke leeches from its two mouths as it affixes you with a gaze of pure malevolence.",
```